ginivision/src/androidTest/java/net/gini/android/vision/test/JpegByteArraySubject.java

```diff
@@ -0,0 +1,120 @@
+package net.gini.android.vision.test;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.google.common.truth.FailureStrategy;
+import com.google.common.truth.Subject;
+import com.google.common.truth.SubjectFactory;
+
+import net.gini.android.vision.test.exif.ExifReader;
+import net.gini.android.vision.test.exif.ExifReaderException;
+
+class JpegByteArraySubject extends Subject<JpegByteArraySubject, byte[]> {
+
+    // Not true that <"subject"> "verb" <"expected">. "failure message".
+    private static final String RAW_MESSAGE_TEMPLATE = "Not true that <%s> %s <%s>. %s.";
+
+    static SubjectFactory<JpegByteArraySubject, byte[]> jpegByteArray() {
+        return new SubjectFactory<JpegByteArraySubject, byte[]>() {
+
+            @Override
+            public JpegByteArraySubject getSubject(final FailureStrategy fs, final byte[] that) {
+                return new JpegByteArraySubject(fs, that);
+            }
+        };
+    }
+
+    JpegByteArraySubject(@NonNull final FailureStrategy failureStrategy,
+            @Nullable final byte[] subject) {
+        super(failureStrategy, subject);
+    }
+
+    void hasUUIDinUserComment(@Nullable  String uuid) {
+        isNotNull();
+        String verb = "has in User Comment UUID";
+
+        if (uuid == null) {
+            fail(verb, (Object) null);
```

Comment:
I'm assuming `verb` is a debug message that will show up in test results?
Also, can you remove the if statement and only have a conditional fail statement?